virtool/startup.py

```diff
@@ -228,6 +229,14 @@ async def init_paths(app: aiohttp.web_app.Application):
         virtool.utils.ensure_data_dir(app["settings"]["data_path"])
 
 
+async def init_postgres(app: aiohttp.web_app.Application):
+    postgres_connection_string = app["config"].get("postgres_connection_string")
```

Comment:
Since the connection string is sourced from the application configuration, which has a default value, we should not use `get()`. Having an unset `postgres_connection_string` would be a bug and we should allow a `KeyError` to be raised if it doesn't exist.

```
app["config"]["postgres_connection_string"]
```